printer_mods/teookie/skr_brace/readme.md

```diff
@@ -0,0 +1,16 @@
+SKR Brace.
+
+Clips your SKR DIN rail mounts together
+- Rigidizes and imobolizes the mounts
```

Comment:
Change `imobolizes` to `immobilizes`

Reply:
Fixed in my fork. Do I have to do something  so that github will roll the changes into this PR?

Reply:
Changes in your fork automatically reflect in this pull request, no need to do anything else for that.